cosi/simulation/cosi_test.go

```diff
@@ -13,6 +14,23 @@ import (
 var tSuite = suites.MustFind("Ed25519")
 
 func TestMain(m *testing.M) {
+	// Raising the FD limit like this might belong in a more central place,
```

Comment:
I suspect this would fail to build with Win32. Checking now via cross-compilation.